src/AmbientSounds.Uwp/Strings/pt-BR/Resources.resw

```diff
@@ -294,4 +294,10 @@
   <data name="Sound-vacuum" xml:space="preserve">
     <value>Vácuo</value>
   </data>
+  <data name="Sound-thunder" xml:space="preserve">
+    <value>Trovões</value>
+  </data>
+  <data name="SettingsNotificationsSwitch.Header" xml:space="preserve">
+    <value>Notificação para novos sons</value>
```

Comment:
I know it might sound crazy, but we should delete most of this file. 

Now that ambie is using the multilingual app toolkit, we don't need to manually edit the `.resw` file anymore. We just need to edit the `.xlf` file. So can you change this file to only contain this:
```xml
<?xml version="1.0" encoding="utf-8"?>
<root>
  <resheader name="resmimetype">
    <value>text/microsoft-resx</value>
  </resheader>
  <resheader name="version">
    <value>2.0</value>
  </resheader>
  <resheader name="reader">
    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
  </resheader>
  <resheader name="writer">
    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
  </resheader>
</root>

```